### PR TITLE
[CI] Fix couchdb e2e trigger and run nightly, part 3

### PR DESCRIPTION
**Pull request by unlikelyzero**
### Describe your changes:
- Fixes the `pr:e2e:couchdb` trigger to register more frequently
- Updates the workflows to run on pushes to master and nightly
- Driveby: reduce confusion with the github annotation reporter

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/nasa/openmct/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/nasa/openmct/pulls) for the same update/change?
* [x] Is this change backwards compatible? For example, developers won't need to change how they are calling the API or how they've extended core plugins such as Tables or Plots.

### Author Checklist

* [x] Changes address original issue?
* [x] Tests included and/or updated with changes?
* [x] Command line build passes?
* [x] Has this been smoke tested?
* [x] Testing instructions included in associated issue OR is this a dependency/testcase change?

### Reviewer Checklist

* [ ] Changes appear to address issue?
* [ ] Reviewer has tested changes by following the provided instructions?
* [ ] Changes appear not to be breaking changes?
* [ ] Appropriate automated tests included?
* [ ] Code style and in-line documentation are appropriate?
* [ ] Has associated issue been labelled unverified? (only applicable if this PR closes the issue)
* [ ] Has associated issue been labelled bug? (only applicable if this PR is for a bug fix)

### CODE REVIEW

File: e2e/playwright-ci.config.js [hunk 1]
```diff
@@ -77,7 +77,6 @@ const config = {
       }
     ],
     ['junit', { outputFile: '../test-results/results.xml' }],
-    ['github'],
```
Comment (unlikelyzero) [inline e2e/playwright-ci.config.js:80]: driveby: this causes too much confusion

File: .github/workflows/pr-platform.yml [hunk 2]
```diff
@@ -1,14 +1,17 @@
 name: 'pr-platform'
 on:
```
Comment (ozyx) [inline .github/workflows/pr-platform.yml:2]: do we need to add an `on` event for the scheduled run? or does it fall under `workflow_dispatch`

File: .github/workflows/pr-platform.yml [hunk 3]
```diff
@@ -1,14 +1,17 @@
...
+  push:
+    branches: master
   workflow_dispatch:
   pull_request:
     types:
       - labeled
       - opened
-
+  schedule:
```
Comment (unlikelyzero) [inline .github/workflows/pr-platform.yml:10]: @ozyx  you mean here?

File: .github/workflows/pr-platform.yml (hunk 2 above)
Comment (unlikelyzero) [reply .github/workflows/pr-platform.yml:2]: https://github.com/nasa/openmct/pull/6782/files/4cd56e48373f409dff148af770a4faf1e33b52ab#r1256537179

**Review by ozyx — APPROVED** [review]
